src/main/kotlin/com/terraformation/backend/nursery/api/BatchesController.kt

```diff
@@ -36,25 +38,35 @@ import org.springframework.web.bind.annotation.RestController
 class BatchesController(
     private val batchStore: BatchStore,
 ) {
+  @ApiResponse(responseCode = "200")
   @ApiResponse404
   @GetMapping("/{id}")
   fun getBatch(@PathVariable("id") id: BatchId): BatchResponsePayload {
     val row = batchStore.fetchOneById(id)
     return BatchResponsePayload(BatchPayload(row))
   }
 
+  @ApiResponse(
+      responseCode = "200",
+      description =
+          "The batch was created successfully. Response includes fields populated by the server, including the batch ID.")
```

Comment:
Can you split these descriptions so they're under the 100-character line length limit? (ktfmt doesn't auto-split long strings.)